docs/CodingStandards.md

```diff
@@ -1,24 +1,22 @@
-# General Formatting
-- Indentation should be 4 spaces instead of tabs
```

Comment:
I think this is covered differently by different files. In general this will be caught by static analysis now.